### ISSUE TITLE
Does not ignore Latex commands if command is preceded by backslash

### ISSUE THREAD

**Issue by MeAmAnUsername** [streetsidesoftware/vscode-spell-checker#1503]
The checker ignores Latex commands (word preceded by `\`), but not if the command is preceded by one or more escaped backslashes (`\\`)
![image](https://user-images.githubusercontent.com/15816455/140461920-0f11ed49-7435-4224-b678-923481f7c97a.png)

Text from the image:
```
askjdh      % Correctly flagged
\askjdh     % Correctly ignored
\\askjdh    % Correctly flagged
\\\askjdh   % Incorrectly flagged
\\\\askjdh  % Correctly flagged
\\\\\askjdh % Not flagged? Seems to be a caching issue
\\\\\askjdg % incorrectly flagged
\\\\\askjdd % incorrectly flagged
```

For five backslashes, it does not give an error for `askjdh` but it does for other incorrect words. Some experimenting brings up the hypothesis that it does not give errors on the same word after a certain number of occurrences. Either way, that's not what this issue is about.

Expected behavior: ignore commands, i.e. ignore words preceded by an odd number of backslashes
Actual behavior: it ignores words only if there is exactly one backslash in front of it
